src/Surfnet/Stepup/Configuration/Value/AllowedSecondFactorList.php

```diff
@@ -19,11 +19,12 @@
 namespace Surfnet\Stepup\Configuration\Value;
 
 use ArrayIterator;
+use Broadway\Serializer\SerializableInterface;
 use IteratorAggregate;
 use JsonSerializable;
 use Surfnet\StepupBundle\Value\SecondFactorType;
 
-final class AllowedSecondFactorList implements JsonSerializable, IteratorAggregate
+final class AllowedSecondFactorList implements JsonSerializable, IteratorAggregate, SerializableInterface
```

Comment:
I don't share this preference: it's OK for me either way. Will try to look into it next time.